README.md

```diff
@@ -1,49 +1,48 @@
 # blue-harvest
 
-## Install
+## Getting Started
 You can npm to install the package:
-`npm install blue-harvest --save-dev`,
-or if you are using Yarn,
-`yarn add blue-harvest -D`
+`npm install https://github.com/copperleaftech/blue-harvest`.
 
 ## Writing Tests
 ### Configurations
-In order for you to use `async await` in the tests, the following line
-```javascript
-SELENIUM_PROMISE_MANAGER: false;
-```
- should be added to the `protractor.conf.js` file.
+In order for you to use `async await` in the tests, `SELENIUM_PROMISE_MANAGER: false` should be added to the `protractor.conf.js` file.
 
 To regulate the resolution of the screenshots, add 
-```javascript
-onPrepare() {
-    browser.driver.manage().window().setSize(1366, 1024);
-    jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
-  }
-```
-in `protractor.conf.js`, inside the `config` block.
+`browser.driver.manage().window().setSize(1366, 1024)`
+ inside `onPrepare()`.
 ### Comparing Screenshots
 To test if a page has changed from a previous screenshot:
 ```
 let result =  await blueharvest.compareScreenshot(
-                   await browser.takeScreenShot(), pathToGolden);
+                   await browser.takeScreenShot(), 'reference.png', 'root/errors');
 ```
-The difference between the two screenshots will be highlighted in pink.
+The difference between the two screenshots will be highlighted in pink. The third parameter defines where to save the difference images and is optional. If the destination path
+for the created difference image is not given but the test fails, `compareScreenshot` will still reject the promise, but will not save the difference image. 
 
 ### Updating Reference Screenshots
 When a change in a page is expected and not a regression, you can update the reference screenshots by
 setting `UPDATE_GOLDENS=1` when you run the tests. In this case, if any test fails,
-blue-harvest will update the reference image with the new one. Only tests involving the 
-reference screenshots (goldens) are required. You can edit the `specs` in `protractor.conf.js` so only necessary tests are run in an update. 
+blue-harvest will update the reference image with the new one. 
 
-In a Linux system, you can simply run `UPDATE_GOLDENS=1 npm run e2e` to update the reference screenshots. 
+In a Linux system, you can simply run `UPDATE_GOLDENS=1 yarn e2e` to update the reference screenshots. 
```

Comment:
Not everyone uses yarn, this might be good to leave as is. Above commands are with npm. Let's switch this back.